### DISCUSSION TITLE
Ability to choose a folder to update images when clicking the "Update screenshot" button

### DISCUSSION THREAD

**Discussion by velosoLB** [FRSOURCE/cypress-plugin-visual-regression-diff#230] [Q&A] [answered]
First and foremost, let me thank you guys for this awesome plugin!

Given the following context:

On my side, I have set up a folder with some reference images (e.g. "reference_images/logo.png", "reference_images/progressBar.png",  "reference_images/form.png" 

When I run a test case, the screenshots are saved in a folder called "visual_testing". This folder is defined through the option "pluginVisualRegressionImagesPath" on Cypress config.

I then compare the images, one by one on my test script using `cy.matchImage({ matchAgainstPath:  'reference_images/logo.png' }) ` and so on...

This works fine when just comparing the images, but when I want to update a reference image using the **"Update screenshot"** button, it doesn't work, because it updates the screenshots on the "visual_testing" folder instead of the ones on the "reference_images" folder.

I was thinking if it was possible to add the functionality of being able to choose a folder when clicking the update screenshot button or by calling a property on the code, just like "matchAgainstPath".

Thank you so much.

**Comment by FRSgit** [comment, marked as answer]
Hey!

Thanks for your message and sorry that it took a while for me to respond.

I'd say that this situation should be considered a bug -> user should expect that the baseline images will get updated when the "Update screenshot" button is clicked, would you agree?

I think having possibility to choose when the "update image" is being saved could be an additional feature but I think it should be added only if somebody comes to us with an understandable use case for it.

**Reply by velosoLB** [reply]
Hello @FRSgit,

Sorry for the late reply. Yes, I think it should be reported as a bug. I've opened an issue here https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff/issues/259 

Best regards
